Improved error message when topic name changes within same supervisor (#13815)
apache/druid: docs/operations/metrics.md

|`ingest/merge/cpu`|Cpu time in Nanoseconds spent on merging intermediate segments.|`dataSource`, `taskId`, `taskType`, `tags`|Depends on configuration. Generally a few minutes at most.|
|`ingest/handoff/count`|Number of handoffs that happened.|`dataSource`, `taskId`, `taskType`, `tags`|Varies. Generally greater than 0 once every segment granular period if cluster operating normally.|
|`ingest/sink/count`|Number of sinks not handoffed.|`dataSource`, `taskId`, `taskType`, `tags`|1~3|

> |`ingest/events/messageGap`|Time gap in milliseconds between the latest ingested event timestamp and the current system timestamp of metrics emission. If this is increasing but lag is low, it likely means that druid is not getting new data. This metric will get reset as new tasks spawn up.|`dataSource`, `taskId`, `taskType`, `tags`|Greater than 0, depends on the time carried in event. |

Reviewer: ```suggestion
|`ingest/events/messageGap`|Time gap in milliseconds between the latest ingested event timestamp and the current system timestamp of metrics emission. If the value is increasing but lag is low, Druid may not be receiving new data. This metric gets reset as new tasks spawn up.|`dataSource`, `taskId`, `taskType`, `tags`|Greater than 0, depends on the time carried in event. |
```

Reviewer 2: "This metric gets reset" --> "This metric is reset"

Reviewer 2: What does this mean? "the current system timestamp of metrics emission." This part is clear: ""the current system timestamp". It is not clear how the "system" (meaning OS) knows about metrics emission. I suspect we just mean `now - __time`. Correct? If so, perhaps reword.

> Druid computes the lag between the current system timestamp and the `__time` value for each record. Druid emits that value, for each record, as this metric. If the value...
